File: LAB-10/Pr2.java

import java.util.Scanner;


public class Pr2 {
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    System.out.println("Enter Size of Array:");
    int n = sc.nextInt();
    int odd = 0, even = 0;
    int[] arr = new int[n];
    System.out.println("Enter Elements of Array:");
    for (int i = 0; i < n; i++) {
      arr[i] = sc.nextInt();
      if (arr[i] % 2 == 0) {
        even++;
      } else {
        odd++;
      }
    }
    System.out.println("Number of Even Elements: " + even);
    System.out.println("Number of Odd Elements: " + odd);
}
    
}

File: LAB-2/Pr2.c
#include <stdio.h>

void main() {
    int n;
    printf("Enter the number of elements in the array:\n");
    scanf("%d", &n);
    int arr[n];
    int sum = 0;
    printf("Enter the elements of the array:\n");
    for (int i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
        sum += arr[i];
    }
    printf("The elements of the array are:\n");
    for (int i = 0; i < n; i++) {
        printf("%d", arr[i]);
    }
    printf("\nSum of the elements of the array is: %d\n", sum);
    printf("Average of the elements of the array is: %d\n", sum / n);
   
}